KsApi/ServiceType.swift

```diff
@@ -144,13 +144,9 @@ public protocol ServiceType {
   func fetchGraphCategory(query: NonEmptySet<Query>)
     -> SignalProducer<CategoryEnvelope, GraphError>
 
-  /// Fetch User's stored cards.
-  func fetchGraphCreditCards(query: NonEmptySet<Query>)
-    -> SignalProducer<UserEnvelope<GraphUserCreditCard>, GraphError>
-
-  /// Fetch a User's account fields
-  func fetchGraphUserAccountFields(query: NonEmptySet<Query>)
-    -> SignalProducer<UserEnvelope<GraphUser>, GraphError>
+  /// Fetches various fields of a given User using graphQL.
+  func fetchGraphUser()
```

Comment:
Thought about that but it would be misleading because we have both a `User` and `GraphUser` object. This particular call is fetching the `GraphUser`.